### DISCUSSION TITLE
Languages

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#305] [Ideas]
Hi,

I would like to contribute to make the Dutch language available. How?

Kind regards, Henk - PH5HP -

**Comment by rejetto** [comment]
Hey, thanks for considering it. Give a look at this
https://github.com/rejetto/hfs/wiki/Translation

**Reply by PH5HP** [reply]
Hi rejetto,

Thank you. I also found it myself. 

Kind regards,
Henk - PH5HP

**Reply by PH5HP** [reply]
Hi Rejetto,

I have almost the translation to Dutch (NL) completed, but I have some row where I am not sure what I can translate to Dutch and what not. Maybe you can help me with that? I did read the information you gave me in the earlier post.

These are some row I am not sure about:

"n_files": "{n,plural,one{# file} other{# files}}",
"n_folders": "{n,plural,one{# folder} other{# folders}}",
"filter_count": "{n,plural, one{# filtered} other{# filtered}}",
"select_count": "{n,plural, one{# selected} other{# selected}}",

"delete_failed": ", {n,plural, one{# failed} other{# failed}}",

"send_files": "Verzonden {n,plural,one{# file} other{# files}}, {size}",


And is this translated OK?
 "send_files": "Verzonden {n,plural,one{# file} other{# files}}, {size}",

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Comment by rejetto** [comment]
your send_files looks quite good, but you are not translating the word "file".
is it ok like it is, or you want to go for "bestand" for example?

another suggestion is: go to Admin/language, and add your file.
You should be able to see the results yourself.

**Reply by PH5HP** [reply]
Hi,

I did not translate it because it was between { } and I thought I that all between { } must not be changed. I did read that somewhere .

So I can change this in all the lines?
- file and files
- folder and folders
- filtered
- selected
- failed
- size

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Comment by PH5HP** [comment]
Hi Rejetto,

I hope I get it right. Attached to this message you will find my Dutch language file ZIPPED.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl
[hfs-lang-nl.zip](https://github.com/rejetto/hfs/files/12216588/hfs-lang-nl.zip)

**Comment by rejetto** [comment]
looks quite good!
What about this?
`        "delete_confirm": "Verwijder {n,plural, one{# item} other{# items}}?",`
is "item" meant?

And, did you try my "another suggestion" ?

**Comment by PH5HP** [comment]
Hi Rejetto,

Item is also used in Dutch ... :-)

Yes, I did add it to the languages and it seems to work ok. I did not see any thing yet which I would alter in something else. One thing I had to do is force HFS to use the Dutch language. For some reason HFS thinks my browser is using the English language. This could be no problem. I mean maybe my browser is English.

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl

**Comment by rejetto** [comment]
your browser may have both languages set but there's also the order to take into account.
you can check it here https://gtranslate.io/detect-browser-language

ok then, your file will be included in next release! thanks for your contribution.
If you'll have the time, you can consider also adding new lines in the future. It doesn't have to be you of course, anyone could do it. Just saying...

**Reply by PH5HP** [reply]
Hi Rejetto,

No problem. I will try to keep the Dutch language up-to-date. :-)

Kind regards,
Henk - PH5HP - ftp.ph5hp.nl
